daemon/create.go

```diff
@@ -129,6 +129,9 @@ func (daemon *Daemon) Create(config *runconfig.Config, hostConfig *runconfig.Hos
 		if err != nil {
 			return nil, nil, err
 		}
+		if err := label.Relabel(v.Path(), container.MountLabel, "z"); err != nil {
```

Comment:
Doesn't matter, if selinux is not supported nothing happens.
Also not any different than what we were doing before interms of selinux in a generic code path.